SequenceBrowser/Widgets/qMRMLSequenceBrowserSeekWidget.cxx

```diff
@@ -57,17 +70,90 @@ void qMRMLSequenceBrowserSeekWidgetPrivate::init()
   Q_Q(qMRMLSequenceBrowserSeekWidget);
   this->setupUi(q);
   this->label_IndexValue->setFont(QFontDatabase::systemFont(QFontDatabase::FixedFont));
-  QObject::connect( this->slider_IndexValue, SIGNAL(valueChanged(int)), q, SLOT(setSelectedItemNumber(int)) );
+  QObject::connect(this->slider_IndexValue, SIGNAL(valueChanged(int)), q, SLOT(setSelectedItemNumber(int)));
   q->updateWidgetFromMRML();
 }
 
+//-----------------------------------------------------------------------------
+QString qMRMLSequenceBrowserSeekWidgetPrivate::formattedIndexValue(int index)
```

Comment:
If it's not too much work, it would be nice to add a test to check formatting with a couple of non-trivial format strings.